src/Microsoft.DotNet.Wpf/src/PresentationBuildTasks/MS/Internal/MarkupCompiler/MarkupCompiler.cs

```diff
@@ -685,6 +685,11 @@ private void GenerateSource()
             GenerateInternalTypeHelperImplementation();
         }
 
+        // adds directrive to top of the file.
+        private static void AddDirectiveToFile(StreamWriter streamWriter, string directive) {
```

Comment:
Code Format issues.

See: https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md

> 1. We use Allman style braces, where each brace begins on a new line. A single line statement block can go without braces but the block must be properly indented on its own line and must not be nested in other statement blocks that use braces (See rule 18 for more details). One exception is that a using statement is permitted to be nested within another using statement by starting on the following line at the same indentation level, even if the nested using contains a controlled block.

```suggestion
        private static void AddDirectiveToFile(StreamWriter streamWriter, string directive) 
        {
```